.github/workflows/_meta.yml

```diff
@@ -81,7 +82,7 @@ jobs:
           compute-release)
             echo "tag=release-compute-$(git rev-list --count HEAD)" | tee -a $GITHUB_OUTPUT
             ;;
-          pr|storage-rc-pr|compute-rc-pr|proxy-rc-pr)
+          pr|storage-rc-pr|compute-rc-pr|proxy-rc-pr|workflow-dispatch)
             BUILD_AND_TEST_RUN_ID=$(gh run list -b $CURRENT_BRANCH -c $CURRENT_SHA -w 'Build and Test' -L 1 --json databaseId --jq '.[].databaseId')
```

Comment:
I'm not sure that it'll work if there's no `Build and Test` workflow run on the branch

Reply:
Yeah, it looks like the same concern is valid for trigger-e2e-tests. Perhaps we need to use $GITHUB_RUN_ID as a fallback?

Reply:
I don't think it'll work for `trigger-e2e-tests` (because they can be triggered on label).

But for `workflow-dispatch` I think we should use `$GITHUB_RUN_ID` (i.e. have a separate statement in case expression)